sphinx/writers/texinfo.py

```diff
@@ -115,7 +115,7 @@ class TexinfoWriter(writers.Writer):
 
     settings_defaults: Dict = {}
 
-    output: str = None
+    output: Optional[str] = None  # type: ignore[assignment]
```

Comment:
false positive. Probably an error in the associated docutils type stubs (i haven't checked).